cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/ConversionHelpers.java

```diff
@@ -45,48 +49,48 @@
 
   private static final Gson GSON = new Gson();
 
-  static Id.Namespace toNamespaceId(String namespace) throws BadRequestException {
+  static NamespaceId toNamespaceId(String namespace) throws BadRequestException {
     try {
-      return Id.Namespace.from(namespace);
+      return new NamespaceId(namespace);
```

Comment:
the changes in this class may cause failures due to https://issues.cask.co/browse/CDAP-5411, like the comment on line 47 indicates